imports/plugins/included/ui-search/lib/containers/searchModalContainer.js

```diff
@@ -272,9 +272,14 @@ function composer(props, onData) {
   Meteor.subscribe("ShopMembers");
   const siteName = getSiteName();
 
-  onData(null, {
-    siteName
-  });
+  const orderSub = Meteor.subscribe("Orders");
```

Comment:
A shop might have tens of thousands of orders, we don't want to pull them all client-side as this will more than likely freeze the client and cause the app to be non-responsive. We've already seen cases where even pull a few hundred orders causes the browser to choke. We need to always be very cautious about what we decide to keep on the client.